apps/app/src/components/Header/HeaderNavigation.tsx

```diff
@@ -16,6 +16,12 @@ export const HeaderNavigation = ({ children }: { children: ReactNode }) => {
 		setIsOpen((prev) => !prev);
 	};
 
+	const handleClickInsideNav = (event: MouseEvent) => {
+		if (event.target instanceof HTMLAnchorElement) {
```

Comment:
A czemu po prostu nie dodać onClick do każdego linków?

Reply:
Myślę, że z poziomu `HeaderNavigation.tsx` mam łatwiejszy dostęp do zmiennej `isOpen` i metody `setIsOpen`. `nav` wrapuje `{children}` więc wewnątrz znaleźć może wszystko, a chcemy zamknąć menu mobilne tylko kiedy przechodzimy między podstronami. Gdybym chciał przekazać zmienną `isHome` i metodę `setIsHome` do dzieci (np. linku) to chyba musiałbym wykorzystać `context`. Ponadto jestem leniwy i chcę, aby to przeglądarka sprawdziła czy klikam w link - dodawanie ręcznie `onClick` do każdego linku jest dość pracochłonne 😜

Reply:
Wykorzystanie context nie ma sensu skoro można by przekazać zwykłą funkcję jako onClick.. Trzeba by natomiast lekko zmienić strukturę.

Twoje rozwiązanie chyba przestanie działać jeśli tylko w środku linka będzie inny element, który zostanie kliknięty?

Reply:
Wprowadziłem poprawki, chciałem uniknąć zmiany struktury, ale okazało się to konieczne i chyba nawet teraz jest lepsza, ponieważ linki przeniesione zostały do `HeaderNavigation`, czyli tam gdzie jest najlepsze dla nich miejsce.